packages/suite-desktop-core/e2e/tests/general/cardano-discovery.test.ts

```diff
@@ -1,19 +1,16 @@
-import { TrezorUserEnvLink } from '@trezor/trezor-user-env-link/src';
-
 import { test, expect } from '../../support/fixtures';
 
-test.beforeAll(async () => {
-    await TrezorUserEnvLink.connect();
-    await TrezorUserEnvLink.startEmu({ wipe: true });
-    await TrezorUserEnvLink.setupEmu({
+test.use({
+    emulatorConf: {
         needs_backup: true,
         mnemonic:
             'cloth trim improve bag pigeon party wave mechanic beyond clean cake maze protect left assist carry guitar bridge nest faith critic excuse tooth dutch',
-    });
+    },
 });
 
-test.afterAll(() => {
-    TrezorUserEnvLink.stopEmu();
+test.beforeEach(async ({ onboardingPage, dashboardPage }) => {
+    await onboardingPage.completeOnboarding();
+    await dashboardPage.discoveryShouldFinish();
 });
```

Comment:
We need to use beforeEach when we want to reuse the instances of the fixtures and objects in the test.